src/kbdfans/kbd19x/kbd19x.json

```diff
@@ -0,0 +1,364 @@
+{
+  "name": "KBD19X",
+  "vendorId": "0x4B42",
+  "productId": "0x0191",
+  "lighting": "qmk_backlight_rgblight",
+  "matrix": {
+    "rows": 8,
+    "cols": 13
+  },
+  "layouts": {
+    "labels": [
+      ["Spacebar", "Default", "7u WKL", "6u Spacebar"],
+      ["Left Shift", "Default", "Split"],
+      ["Backspace", "Default", "Split"],
+      ["Numpad", "Default", "2u Enter", "1Us"]
+    ],
+
+    "keymap": [
+    [
+    {
+      "x": 15.75,
+      "c": "#aaaaaa"
+    },
+    "7,00\n\n\n2,1",
```

Comment:
QMK VIA keymap uses Layout_ansi. 7,00 for split backspace not in Layout_ansi in QMK. May want to change QMK VIA keymap to use Layout_all.